packages/x1e80100-linux.nix

```diff
@@ -21,11 +19,14 @@ linuxPackagesFor (buildLinux {
       git config user.name "nix"
       git config user.email "nix"
 
-      git fetch 'https://git.codelinaro.org/stephan.gerhold/linux.git' be611e02a1d79ffed11c0bd969ac4a08c0367438
-
       # EL2 improvements by Stephan Gerhold
+      git fetch 'https://git.codelinaro.org/stephan.gerhold/linux.git' be611e02a1d79ffed11c0bd969ac4a08c0367438
       git cherry-pick 4a9b759f28636aa25db450062fd9453511d530d9..be611e02a1d79ffed11c0bd969ac4a08c0367438
 
+      # Surface Pro 11 device tree + fixes by Dale Whinham
+      git fetch 'https://github.com/dwhinham/kernel-surface-pro-11.git' 9bf2b621465029671d73374ef6495e4a81b70e80 --depth 15
+      git cherry-pick 5f9a04a612b34dafe906da842166c2efd820f589..9bf2b621465029671d73374ef6495e4a81b70e80 --allow-empty --empty=drop
```

Comment:
Looked through these, `5f9a04a612b34dafe906da842166c2efd820f589..9bf2b621465029671d73374ef6495e4a81b70e80` seems like a very reasonable set of commits to include. I expect that we will want to update this range occasionally, but future auditing should be quite fast using `git range-diff`.